bucket/filelight.json

```
@@ -22,16 +22,15 @@
         ]
     ],
     "checkver": {
-        "url": "https://apps.kde.org/filelight",
-        "regex": "filelight-([\\d.]+)-(?<build>[\\d]+)-windows-(?<lib>\\w+)-cl\\.exe"
+        "url": "https://apps.kde.org/de/filelight/index.xml",
+        "regex": "Release ([\\d\\.]+) of Filelight"
```

```suggestion
        "url": "https://binary-factory.kde.org/job/Filelight_Release_win64/lastSuccessfulBuild/artifact/",
        "regex": "filelight-([\\d.-]+)-windows-msvc(?<msvcver>\\d+)_64-cl\\.7z"
```

The version went from 22.04.3-**1064** to 22.04.3-**1065*.
We should include `-1065` as the version, so that we can track minor updates